README.md

```diff
@@ -15,7 +15,7 @@ Since `3.3.x`, this repository also contains `reactor-tools`, a java agent aimed
 
 ## Getting it
    
-**Reactor 3 requires Java 8 or + to run**.
+**The Minimum requirement for Reactor 3 is Java 8 or higher**.
```

Comment:
Thanks for the suggestion. I'm not a native speaker so I might not know best, but it feels like the proposed sentence is not as concise and the caps/non-caps are not consistent. Why would "Minimum" be capitalized? To be honest, the current flow of the sentence makes more sense to me. Would "Reactor 3 requires Java 8 or higher to run" be acceptable? This way, only the "+" sign is replaced with "higher" and maintains the conciseness.